docs/source/tutorial/queries.mdx

```diff
@@ -428,6 +468,18 @@ export default Profile;
 
 </MultiCodeBlock>
 
-If you try to render this query, you'll notice that it returns null. This is because we need to implement our login feature first. We're going to tackle login in the next section.
+You should recognize all of the concepts in this code from the pages we've already completed, with one highlighted exception: we're setting a `fetchPolicy`.
+
+### Customizing the fetch policy
+
+As mentioned earlier, Apollo Client stores query results in its cache. If you query for data that's already present in your cache, Apollo Client can return that data without needing to fetch it over the network.
+
+_However_, cached data can become stale. Slightly stale data is acceptable in many cases, but we definitely want our user's list of booked trips to be up to date. To handle this, we've specified a **fetch policy** for our `GET_MY_TRIPS` query.
+
+A fetch policy defines how Apollo Client uses the cache for a particular query. The default policy is `cache-first`, which means Apollo Client checks the cache to see if the result is present before making a network request. If the result is present, no network request occurs.
+
+By setting this query's fetch policy to `network-only`, we guarantee that Apollo Client _always_ queries our server to fetch the user's most up-to-date list of booked trips.
+
+> For a list of all support fetch policies, see [Configuring fetch logic](https://www.apollographql.com/docs/react/data/queries/#configuring-fetch-logic).
```

Comment:
Not sure if this bears mentioning here, but `nextFetchPolicy` is a good way to switch back to `cache-first` after an initial `network-only` or `cache-and-network` fetch.

Reply:
Ah yeah, that definitely bears mentioning in the dedicated fetch policy section of the docs at least